vcd/resource_vcd_network_routed.go

```diff
@@ -263,10 +260,17 @@ func resourceVcdNetworkRead(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
+func resourceVcdNetworkDeleteLocked(d *schema.ResourceData, meta interface{}) error {
+	lockParentEdgeGtw(d)
+	defer unLockParentEdgeGtw(d)
+
+	return resourceVcdNetworkDelete(d, meta)
+}
+
 func resourceVcdNetworkDelete(d *schema.ResourceData, meta interface{}) error {
 	vcdClient := meta.(*VCDClient)
-	vcdClient.Mutex.Lock()
-	defer vcdClient.Mutex.Unlock()
+
+	time.Sleep(10 * time.Second)
```

Comment:
Why this ten second sleep here and even at the very beginning of the function??

Reply:
Leftover from my investigation, removed